swagger.yaml

```diff
@@ -1543,8 +1529,30 @@ parameters:
       Limit the the number of items to be returned per page.
       The 'limit' parameter is typically used with the 'offset' parameter to facilitate pagination.
 
-  baseDate:
-    name: baseDate
+  start:
+    name: start
+    in: query
+    required: false
+    type: string
+    format: 'date-time'
+    x-example: 2014-04-17
+    description: >-
+      Limit the entities to ones within the specified range.
+      The 'end' parameter is typically used in combination with the 'start' param to form a date range.
```

Comment:
Are the dates inclusive so end would include all the events on that date if it has no 'time' included; or if it has a time, it includes events that fall on that time?

Reply:
Can't remember. Probably plugged straight into a between clause so >= start and < end